### PR TITLE
Create missing OH rep yamls

### PR DESCRIPTION
**Pull request by OdeclasV**
Creating couple of yamls of OH Reps that I missed last week

### CODE REVIEW

File: members/SOHL4908448c.yaml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+            TN: "TN"
+            TX: "TX"
+            UT: "UT"
+            VT: "VT"
+            VA: "VA"
+            WA: "WA"
+            WV: "WV"
+            WI: "WI"
+            WY: "WY"
+    - click_on:
```
Comment (tomdionesotes) [inline members/SOHL4908448c.yaml]: do we need any waits or finds?

Comment (OdeclasV) [reply members/SOHL4908448c.yaml]: I don't think so. Tested these forms a few times to see if there was an issue related to waits or finds but didn't run into any. I could add a them tho, for extra precaution if we think it'd be valuable

Comment (tomdionesotes) [reply members/SOHL4908448c.yaml]: Nah, all good!

**Comment by OdeclasV** [comment]
I found more OH targets without yamls. Thought it was just a couple that I had missed but it might be more than I expected. Going to do a more thorough review before merging this.

**Comment by OdeclasV** [comment]
Yeap, found way more missing OH yamls. These yamls are identical to the two I previously committed in this PR, fyi @tomdionesotes

**Comment by tomdionesotes** [comment]
@OdeclasV good to go!

**Comment by OdeclasV** [comment]
gracias!

**Pull request merged**